samples/bluetooth/shell_bt_nus/README.rst

```diff
@@ -15,35 +15,49 @@ See :ref:`shell_bt_nus_host_tools` for more information about the host tools tha
 Requirements
 ************
 
-* One of the following development boards:
+The sample supports the following development kits:
 
-  * |nRF5340DK|
-  * |nRF52840DK|
-  * |nRF52DK|
-
-* A second nRF52 Development Kit board (PCA10040) for connecting with bt_nus_shell.py.
-  Alternatively, you can use :ref:`ble_console_readme` for connecting (Linux only).
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set1_start
+   :end-before: set1_end
 
+You also need an additional nRF52 DK (PCA10040) for connecting with ``bt_nus_shell.py``.
+Alternatively, you can use :ref:`ble_console_readme` for connecting, using Linux only.
 
 Building and running
 ********************
 
-This sample can be found under :file:`samples/bluetooth/shell_bt_nus` in the
-|NCS| folder structure.
-
-See :ref:`gs_programming` for information about how to build and program the
-application.
+.. |sample path| replace:: :file:`samples/bluetooth/shell_bt_nus`
 
-Testing
-=======
+.. include:: /includes/build_and_run.txt
 
-After programming the sample to your board, test it by connecting to it and sending some shell commands.
+Testing using shell_bt_nus
+==========================
 
 .. include:: ../../../include/shell/shell_bt_nus.rst
-   :start-after: testing_start
-   :end-before: testing_end
+   :start-after: testing_bt_nus_shell_intro_start
+   :end-before: testing_bt_nus_shell_intro_end
+
+After programming the sample to your development kits, test it by performing the following steps:
+
+1. Start a terminal emulator like PuTTY and connect through UART to the ``shell_bt_nus`` application running on the development kit to check the log.
+   See :ref:`gs_testing` for more information on how to connect with PuTTY through UART.
+#. On the host PC, install *Python 2*.
+#. Run the following command to install the |BLE| driver: ``pip2 install pc_ble_driver_py``.
+#. Run the following command to install :ref:`nRF Util`: ``pip2 install --user nrfutil==4.0.0``.
+#. Connect the nRF52 board meant for connecting with ``bt_nus_shell.py`` to your PC.
+#. Start the ``bt_nus_shell.py`` script with the correct parameters, for example::
+
+       bt_nus_shell.py --name BT_NUS_shell --com COM237 --family NRF52 --snr 682560213
+#. Open a terminal emulator like PuTTY and set the Connection Type to ``Raw`` and the  Destination Address to ``127.0.0.1:8889``.
+#. Press Enter in the terminal.
```

Comment:
Is this Enter or Open?

Reply:
I think it is actually both, press guilabel-open to actually open the terminal, then press enter in the terminal.
@nordic-krch Could you check this one?

Reply:
it's pressing `enter`

Reply:
ok, so I rephrased it